assets/src/blocks/Cookies/CookiesFieldResetButton.js

```diff
@@ -0,0 +1,28 @@
+const {__} = wp.i18n;
+
+import {Tooltip} from '@wordpress/components';
+
+const COOKIES_DEFAULT_COPY = window.p4bk_vars.cookies_default_copy || {};
```

Comment:
The `cookies_default_copy` is set from [blocks plugin](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/classes/class-loader.php#L460), we need to move it in master-theme.

Reply:
Ah yes, good point 👍 On it!